cnames_active.js

```diff
@@ -1036,6 +1036,7 @@ var cnames_active = {
   "feeble": "feeblejs.github.io/feeble",
   "fela": "cname.vercel-dns.com", // noCF
   "fem": "frani.github.io/fem",
+  "femd": "femarkdown.github.io"
```

Comment:
```suggestion
  "femd": "femarkdown.github.io",
```
As the CI error notes, you are missing a comma at the end of your entry